test/style.js

```diff
@@ -53,3 +53,10 @@ test('vendor prefixed styles are correctly camel-cased', function (t) {
   t.plan(1)
   t.equal(el.style.WebkitTransition, 'opacity 100ms ease')
 })
+
+test('pascal-cased, vendor prefixed styles are not camel-cased', function (t) {
+  var el = mk().node()
+  el.setAttribute('style', 'WebkitTransition: opacity 100ms ease')
```

Comment:
I think it's maybe not enough. As specified by React, the vendor prefixes _except_ `ms` should be pascal cased. So that'd make sense to check for the substring? That use case should be added to the test as well :8ball: